includes/class-genesis-simple-hooks.php

```diff
@@ -0,0 +1,230 @@
+<?php
+/**
+ * This file handles the creation of the Simple Hooks admin menu.
+ *
+ * @package genesis-simple-hooks
+ */
+
+/**
+ * Plugin Name: Genesis Simple Hooks
+ * Plugin URI: http://www.studiopress.com/plugins/simple-hooks
+ *
+ * Description: Genesis Simple Hooks allows you easy access to the 50+ Action Hooks in the Genesis Theme.
+ *
+ * Author: StudioPress
+ * Author URI: http://www.studiopress.com/
+ *
+ * Version: 2.2.1
+ *
+ * Text Domain: genesis-simple-hooks
+ * Domain Path: /languages
+ *
+ * License: GNU General Public License v2.0 (or later)
+ * License URI: http://www.opensource.org/licenses/gpl-license.php
+ */
+class Genesis_Simple_Hooks {
+
+	/**
+	 * Plugin version
+	 *
+	 * @var $plugin_version The plugin version
+	 */
+	public $plugin_version = GENESIS_SIMPLE_HOOKS_VERSION;
+
+	/**
+	 * Minimum WordPress version.
+	 *
+	 * @var $min_wp_version The minimum WordPress version
+	 */
+	public $min_wp_version = '4.7.2';
+
+	/**
+	 * Minimum Genesis version.
+	 *
+	 * @var $min_wp_version The minimum WordPress version
+	 */
+	public $min_genesis_version = '2.5.0';
+
+	/**
+	 * The plugin textdomain, for translations.
+	 *
+	 * @var $plugin_textdomain The plugin text domain
+	 */
+	public $plugin_textdomain = 'genesis-simple-hooks';
+
+	/**
+	 * The url to the plugin directory.
+	 *
+	 * @var $plugin_dir_url The plugin directory url
+	 */
+	public $plugin_dir_url;
+
+	/**
+	 * The path to the plugin directory.
+	 *
+	 * @var $plugin_dir_path The plugin directory path
+	 */
+	public $plugin_dir_path;
+
+	/**
+	 * The main settings field for this plugin.
+	 *
+	 * @var $settings_field The minimum settings field
+	 */
+	public $settings_field = 'simplehooks-settings';
+
+	/**
+	 * Admin menu and settings page.
+	 *
+	 * @var $admin The minimum WordPress version
+	 */
+	public $admin;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 2.2.0
+	 */
+	public function __construct() {
+
+		$this->plugin_dir_url  = plugin_dir_url( __FILE__ );
```

Comment:
@ethan-schaffer The plugin root level `GENESIS_SIMPLE_HOOKS_DIR` and `GENESIS_SIMPLE_HOOKS_URl` should be used here. This will probably need some adjustments within the class and possibly other classes.

Reply:
I just addressed this, thank you for pointing it out!